measuring_ci/costs.py

```diff
@@ -0,0 +1,24 @@
+import pandas
+
+from measuring_ci.files import open_wrapper
+
+
+def fetch_worker_costs(csv_filename):
+    """static snapshot of data from worker_type_monthly_costs table."""
+
+    with open_wrapper(csv_filename, 'r') as f:
```

Comment:
pandas supports s3 directly, so we should be able to `import boto3` and just `.read_csv('s3://...')`